tests/postgres/generated_test.go

```diff
@@ -0,0 +1,15 @@
+package postgres
+
+import (
+	. "github.com/go-jet/jet/v2/tests/.gentestdata/jetdb/test_generated/table"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
```

Comment:
This tests are intended to be used with a live db. For generated column tests we can write a tests to test simple SELECT, INSERT or UPDATE query. You can then test mutable columns by comparing `.Sql()` or `.DebugSql()` query using `testutils.AssertStatementSql` or `testutils.AssertDebugStatementSql`.  For example - [check](https://github.com/go-jet/jet/blob/ce3c622fa4f8b6f7a18a8c84a386f710e6bef430/tests/postgres/sample_test.go#L17).
Result of the db query is not of importance, just error check would be enough.

The test can be added in existing `sample_test.go`.

Reply:
SQL and test migrated and updated.